src/DependencyInjection/Compiler/EntityServiceCompilerPass.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
If we just want to injected tagged services we do not need a custom CompilerPass

https://symfony.com/doc/4.4/service_container/tags.html#reference-tagged-services

Reply:
Tks, I learned something